<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Countdown Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1900</width>
         <height>932</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="tileGridWidget">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="createFrame">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="titleInput">
         <property name="placeholderText">
          <string>Enter Event Title</string>
         </property>
        </widget>
       </item>
 	<item>
  	<widget class="QCheckBox" name="unhideCheckBox">
   	<property name="text">
    	<string>Count Up</string>
   	</property>
  	</widget>
 	</item>
 	<item>
  	<widget class="QCheckBox" name="blackTextCheckBox">
   	<property name="text">
    	<string>Black Text</string>
   	</property>
  	</widget>
 	</item>
       <item>
        <widget class="QComboBox" name="yearCombo"/>
       </item>
       <item>
        <widget class="QComboBox" name="monthCombo"/>
       </item>
       <item>
        <widget class="QComboBox" name="dayCombo"/>
       </item>
       <item>
        <widget class="QComboBox" name="hourCombo"/>
       </item>
       <item>
        <widget class="QPushButton" name="chooseImageButton">
         <property name="text">
          <string>Select Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="text">
          <string>Create Event</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="plusButton">
      <property name="styleSheet">
       <string notr="true">border-radius: 30px; background-color: #0078D4; color: white; font-size: 28pt; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editModeButton">
      <property name="text">
       <string>Edit Mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
